Rmarkdown/Rmarkdown-intro.Rmd

```diff
@@ -0,0 +1,643 @@
+
+## Equations
+
+Equations are embedded in a pair of dollar symbols.
+Rstudio will show a preview of the equation as you type it.
+Equations are written with [LaTeX notation](https://en.wikibooks.org/wiki/LaTeX/Mathematics).
+
+```{r equations, echo = FALSE}
+read_csv('
+What, How, Output         
+"Lower-case Greek letters", "`$\\sigma$`", "$\\sigma$"
+"Upper-Case Greek Letters", "`$\\Sigma$`", "$\\Sigma$"
+"Subscript", "`$\\beta_{0}$`", "$\\beta_{0}$"
+"Superscript", "`$\\chi^{2}$`", "$\\chi^{2}$"
+"Fractions", "`$\\frac{1}{2}$`", "$\\frac{1}{2}$"
+"Roots", "`$\\sqrt{4} = 2$`", "$\\sqrt{4} = 2$"') %>% 
+  kbl(booktabs = TRUE) %>% 
+  kable_styling(full_width = TRUE)
+```
+
+<br>
+
+Several of these elements can be put together to make the equation of a standard deviation.
+The double dollar enclosure gives the equation its own line.
+When making complex formula, build one element at a time, often starting in the middle, rather than trying to get it all working at once.
+
+`$$SD = \sqrt{\frac{\sum_{i=1}^{n}{(x_i - \bar{x})^2}}{n-1}}$$` $$SD = \sqrt{\frac{\sum_{i=1}^{n}{(x_i - \bar{x})^2}}{n-1}}$$
+
+# Towards a manuscript
+
+## Citations and bibliographies
+
+R markdown can add citations and a bibliography to a document.
+Use the bibliography field in the YAML to point to the bibliography file.
+
+```{r bibliography, echo = FALSE, eval = FALSE}
+---
+title: "My Manuscript"
+output: html_document
+bibliography: reproducibility.bib
+---
+```
+
+```{r bibliography-flair, echo=FALSE}
+decorate("bibliography", eval = FALSE) %>% 
+  flair_lines(4)
+```
+
+The bibliography file can be in one of several formats, including bibTeX (\*.bib) and endnote (\*.enl).
+Any reference manager should be able to export to a bibTeX file.
+
+### Bibliography placement
+
+By default, the bibliography is placed at the end of the document.
+It is a good idea to end the R markdown file with an appropriate header.
+
+```{r bib-pos, echo = FALSE, eval = FALSE}
+# Conclusions
+
+R markdown is cool.
+
+# References {-}
+```
+
+```{r bib-pos-flair, echo=FALSE}
+decorate("bib-pos", eval = FALSE) %>% 
+  flair_lines(5)
+```
+
+The `{-}` stops R markdown from numbering the reference section header.
+
+If you want to move the bibliography, for example because you need to finish with an appendix, you need to use this bit of magic.
+You also need to use output format `bookdown::html_document2` in the YAML.
+
+```{r bib-move, echo = FALSE, eval = FALSE}
+# Conclusions
+
+R markdown is very cool.
+
+# References {-}
+
+&lt;div id="refs"&gt;&lt;/div&gt;
+
+# Appendix A
+```
+
+```{r bib-move-flair, echo=FALSE}
+decorate("bib-move", eval = FALSE) %>% 
+  flair_lines(7)
+```
+
+### Citations
+
+Use the citation keys from the bibliography to cite the reference in the text.
+So to cite the reference with the citation key "Cooper2017" we would use.
+
+
+    Reproducible code [@Cooper2017] ...
+    @Cooper2017 discuss ... 
```

Comment:
Is [-@Cooper2017] doing the same thing?

Reply:
no. [-@Cooper2017]  will render as (2017). Useful occasionally. When I started I didn't realise I could do @Cooper2017 so would write Cooper et al [-@Cooper2017]